sample_deploy_config.json

```diff
@@ -1,16 +1,19 @@
 {
   "project_name": "<YOUR PROJECT NAME HERE>",
   "stack_name": "<YOUR STACK NAME HERE>",
+  "//": ["checkov:skip=CKV_SECRET_6: ignore stack_api_key_env in this file"],
   "stack_api_key_env": "IBMCLOUD_API_KEY",
   "config_order": [
     "1 - Account Infrastructure Base",
     "2a - Security Service - Key Management",
-    "2b - Security Service - Secret Manager",
-    "2c - Security Service - Security Compliance Center",
-    "3 - Observability - Logging Monitoring Activity Tracker",
-    "4 - WatsonX SaaS services",
-    "5 - Sample RAG app - Application Lifecycle Management",
-    "6 - Sample RAG app configuration"
+    "2b - WatsonX SaaS services",
```

Comment:
watson should be after 5b

Reply:
stackDeploy.py is not used or needed anymore, since projects supports auto deploy. will be removed